libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/dh-market-participant-edit-organization.component.html

```diff
@@ -35,37 +35,28 @@ <h1 *ngIf="!isEditing">{{ t("organizationCreateHeader") }}</h1>
     <div class="spinner-container" *ngIf="isLoading">
       <watt-spinner></watt-spinner>
     </div>
-    <ng-container *rxLet="contacts$ as contacts">
-      <ng-container *rxLet="organization$ as organization">
-        <div [hidden]="isLoading">
-          <!-- Tabs -->
-          <watt-tabs class="tabs">
-            <watt-tab [label]="t('masterDataTabHeader')">
-              <dh-market-participant-organization-master-data
-                [organization]="organization"
-                (hasChanges)="onMasterDataChanged($event)"
-              ></dh-market-participant-organization-master-data>
-            </watt-tab>
-
-            <watt-tab [label]="t('contactDataTabHeader')">
-              <dh-market-participant-organization-contact-data
-                [contacts]="contacts"
-                (contactsChanged)="onContactsChanged($event.add, $event.remove)"
-              ></dh-market-participant-organization-contact-data>
-            </watt-tab>
-          </watt-tabs>
-
-          <!-- Confirmation  -->
-          <div class="button-container">
-            <watt-button variant="secondary" (click)="onCancelled()"
-              >{{ t("cancelLabel") }}
-            </watt-button>
-            <watt-button variant="primary" (click)="onSaved()">{{
-              t("saveLabel")
-            }}</watt-button>
-          </div>
+    <ng-container *rxLet="organization$ as organization">
```

Comment:
Nitpick: Consider using `template-binding` and remember to handle the error states as well. 
https://github.com/rx-angular/rx-angular/blob/main/libs/template/docs/api/let-directive.md#using-the-template-binding

Reply:
My comment was regarding using `rxError`, `rxSuspense` and `rxComplete` to make it easier to handle the different states of the template.